.github/workflows/ci.yml

```diff
@@ -60,34 +60,6 @@ jobs:
           version: latest
           args: buildAll
 
-      - name: Check for E2E
```

Comment:
why changes in github workflow files ? Have they not been generated by grafana plugin generator tool ?

Reply:
See answer below. 
But good point, instead of having a seperate e2e workflow I will include this here. I will check how the ci file that is generated today with the tool looks like and copy it over.